jobsdb/integration_test.go

```diff
@@ -1093,8 +1094,9 @@ func TestJobsdbSanitizeJSON(t *testing.T) {
 		{
 			string(TEXT),
 			[]testCase{
-				{`\u0000`, `\u0000`, nil},
-				{`\u0000☺\u0000b☺`, `\u0000☺\u0000b☺`, nil},
+				{"\x00", "", nil},
+				{`\u0000`, "", nil},
```

Comment:
this change now happens because of the invalid character in user_id column.
It might be prudent to have some of those previous cases too

Reply:
If you notice, now TEXT column type exhibits the same behaviour as JSONB column